examples/server.mjs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+import { load } from "@azure/app-configuration-provider";
+import { DefaultAzureCredential } from "@azure/identity";
+const endpoint = process.env.APPCONFIG_ENDPOINT;
+const credential = new DefaultAzureCredential();
+const appConfig = await load(endpoint, credential, {
+    refreshOptions: {
+        enabled: true,
+        refreshIntervalInMs: 5_000
+    }
+});
+
+appConfig.onRefresh(() => {
+    console.log("Configuration has been refreshed.");
```

Comment:
In a real example, we won't have a fake log but something like `appConfig.constructConfigurationObject()` here. That's how this callback is supposed to be used.